webapp/src/Controller/API/ProblemController.php

```diff
@@ -180,6 +180,11 @@ public function listAction(Request $request): Response
      *                 property="problem",
      *                 description="Optional: problem id to update.",
      *                 type="string"
+     *             ),
+     *             @OA\Property(
+     *                 property="delete_old_data",
```

Comment:
`data` (and `old`) (also in other places in this PR, is a bit vague, maybe something like 'clear existing testcases'?

Reply:
I don't like 'testcases' since it also deletes attachments and other stuff like limits and problem texts.

Reply:
Alright, let's not fess over it too much then